code/dqc/datechk.q

```diff
@@ -1,10 +1,8 @@
 \d .dqc
-datechk:{[dirname]                                                                                              /- function to check date vector contains latest date in an hdb
-  system"l ",dirname;
+datechk:{[]                                                                                                     /- function to check date vector contains latest date in an hdb
   if[not `PV in key`.Q;
-    .lg.o[`datechk;"The directory is not partitioned"]; :0b];
```

Comment:
Should keep TorQ logging in this function to aid debugging. Would be good to give more details for the log - so include what directory has the issue.